repositories.yaml

```diff
@@ -611,3 +611,27 @@ applications:
     channels:
       - v1_name: mozilla-vpn
         app_id: mozillavpn
+
+  - app_name: rally_study_zero_one
+    canonical_app_name: Rally Study-01
+    app_description: |
+      This is an exploratory study on what sites people
+      use and how long they use them, for the Rally
+      platform. It is implemented as a cross-browser
+      WebExtension.
+    url: https://github.com/mozilla-rally/rally-study-01
+    notification_emails:
+      - than@mozilla.com
+      - hulmer@mozilla.com
+    branch: master
+    metrics_files:
+      - metrics.yaml
+    ping_files:
+      - pings.yaml
+    dependencies:
+      - glean-js
+    channels:
+      - v1_name: rally-study-zero-one
+        app_id: rally-study-zero-one
```

Comment:
I'll just note that `rally_zero_one` and `rally_study_zero_one` are close enough together to be very confusing. Why not `rally_zero_two`; is this the same study?

Reply:
The study itself is called "Rally 01", so calling it "zero two" would be even more confusing :) However, we're apparently hitting a GCP limit with this name and we're [discussing potential solutions here](https://bugzilla.mozilla.org/show_bug.cgi?id=1713416).